pkg/agent/loadbalancer/loadbalancer_test.go

```diff
@@ -84,7 +84,7 @@ func assertNotEqual(t *testing.T, a interface{}, b interface{}) {
 	}
 }
 
-func TestFailOver(t *testing.T) {
+func Test_FailOver(t *testing.T) {
```

Comment:
Why are we renaming all of the tests?

Reply:
It is explained more above, but basically its to help separate and distinguish between unit and integration tests. You can now run just integration tests with a call to `go test ./pkg/... -run TestIntegration_` and only unit tests with a call to `go test ./pkg/... -run Test_`. It was either establish a consistent and separate naming scheme or use build tags to enable separate testing.

Reply:
I wonder if it'd be clearer and easier to have Test_UnitFailover and Test_IntegrationSomething so we can simplify the exec of the tests ie `go test ./pkg/... -run Unit` and `go test ./pkg/... -run Integration`

Reply:
I will implement this suggestion. Personally I'm not attached with a specific name order, as long as we can easily distinguish between the Unit tests and Integration tests by name.